indico/modules/events/contributions/templates/management/contribution_person_list.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'events/management/_event_person.html' %}
+
+{% block no_persons_message %}
+    <td colspan="6">{% trans %}No persons in the selected contributions yet{% endtrans %}</td>
```

Comment:
"There are no persons associated with the selected contributions." sounds better IMO